pipelines/pipeline_navvis_rig.py

```diff
@@ -0,0 +1,305 @@
+            # the QR code across multiple tiles.
+            tiles_format_qrcode = "none"
+            run_navvis_to_capture.run(
+                navvis_path, capture_qrcode, tiles_format_qrcode
+            )
+
+            # Reload capture_qrcode, so we have access to the copied meshes.
+            capture_qrcode = Capture.load(capture_qrcode_path)
+
+            # Detect QR codes in the session.
+            run_qrcode_detection_session(
+                capture_qrcode, session, mesh_id, **kargs
+            )
+
+            # Copy QR code output from capture_qrcode to capture.
+            shutil.copytree(
+                str(capture_qrcode.proc_path(session) / "qrcodes"),
+                str(capture.proc_path(session) / "qrcodes"),
+                dirs_exist_ok=True,
+            )
+
+        if visualization:
+            to_meshlab_visualization.run(
+                capture,
+                session,
+                f"trajectory_{session}",
+                export_mesh=True,
+                export_poses=True,
+                mesh_id=mesh_id,
+            )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        description=description, formatter_class=argparse.RawTextHelpFormatter
+    )
+    parser.add_argument(
+        "--input_path",
+        type=Path,
+        required=True,
+        help="Specifies NavVis data path. Inside this path, there should be "
+        "a folder for each session. Each session folder should be the output "
+        "of the NavVis data conversion tool (proc/ folders).",
+    )
+    parser.add_argument(
+        "--output_path",
+        type=Path,
+        required=False,
+        default=None,
+        help="Output path. Default: current working directory. If the path "
+        "does not exist, it will be created.",
+    )
+    parser.add_argument(
+        "--tiles_format",
+        type=str,
+        required=False,
+        default="3x3",
+        choices=TILE_CHOICES,
+    )
+    parser.add_argument(
+        "--sessions",
+        nargs="*",
+        type=str,
+        required=False,
+        default=None,
+        help="[Optional] List of sessions to process. Useful when "
+        "processing only specific sessions.",
+    )
+    parser.add_argument(
+        "--use_simplified_mesh",
+        action=argparse.BooleanOptionalAction,
+        required=False,
+        default=False,
+        help="Use simplified mesh. Default: False. Pass --use_simplified_mesh "
+        "to set to True. This is useful for large scenes.",
+    )
+    parser.add_argument(
+        "--meshing_method",
+        type=str,
+        required=False,
+        default="advancing_front",
+        choices=["advancing_front", "poisson"],
+        help="Meshing method. Default: advancing_front.",
+    )
+    parser.add_argument(
+        "--meshing",
+        action=argparse.BooleanOptionalAction,
+        required=False,
+        default=True,
```

Comment:
we always do meshing? shouldn't we do it False? meshing takes a long time, right? or is this just running the marching cube?

Reply:
This is a full pipeline, make senses to select the defaults we want to have. The individual scripts can be called separately, like: `scantools.run_navvis_to_capture.py`